### ISSUE TITLE
DialogueFlow app integration fails to load credentials

### ISSUE THREAD

**Issue by Shalashtein** [chatwoot/chatwoot#6908]
**Describe the bug**

When a user sends a message through the website channel chatwoot raises an error while trying to load the key file. More info and stack trace below

**To Reproduce**

1. Setup the website channel
2. Use the service account and dialogflow to fetch the key file
3. Setup dialogueflow integration with the website channel
4. Send a message to the website channel

**Expected behavior**

Chatwoot should successfully connect to google and send the response from dialogueflow to the user

**Sentry Log**

[Log](https://hybr-f6.sentry.io/share/issue/1adbe1bc4ec346228ed2a73400b1c635/)

**Environment**

Describe whether you are using Chatwoot Cloud (app.chatwoot.com) or a self-hosted installation of Chatwoot. If you are using a self-hosted installation of Chatwoot, describe the type of deployment (Docker/Linux VM installation/Heroku/Kubernetes/Other).

- [ ] app.chatwoot.com (Chatwoot Cloud)
- [X] Self-hosted
- - [X] Linux VM
- - [ ] Docker
- - [ ] Kubernetes
- - [ ] Heroku
- - [ ] Other (Please specify)

**Cloud Provider** (If applicable)
- [X] AWS
- [ ] GCP
- [ ] Azure
- [ ] DigitalOcean
- [ ] Others

**Comment by pranavrajs** [comment]
@Shalashtein Can you please confirm that the permissiona are set properly as per [this document](https://www.chatwoot.com/docs/product/channels/live-chat/integrations/dialogflow/)?

**Comment by Shalashtein** [comment]
Yes I can confirm that the service account has the Dialogflow API Client permissions, and I've added all the dialogFlow permissions and tried recreating the service account without luck

**Comment by stonecharioteer** [comment]
This issue is being closed due to lack of activity for over a year. [For more details, visit this discussion.](https://github.com/orgs/chatwoot/discussions/12810)
